User/UserRepositoryCollection.php

```diff
@@ -91,7 +91,7 @@ public function supportsClass($className)
     /**
      * @param $token
      *
-     * @return UserInterface|null
+     * @return null|User|void
```

Comment:
is dat niet hetzelfde als gewoon null? ik zou void gewoon weglaten, want je zet bij een method die niks returned toch ook niet `@return void`?